install/traf_tools_setup.sh

```diff
@@ -0,0 +1,249 @@
+protoc --version
+
+# install bison
+cd $BASEDIR
+wget http://ftp.gnu.org/gnu/bison/bison-3.0.tar.gz
+tar -xzf bison-3.0.tar.gz
+cd bison-3.0
+./configure --prefix=$TOOLSDIR/bison_3_linux
+make
+make check
+make install
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of BISON"
+fi
+
+# install udis
+cd $BASEDIR
+wget http://sourceforge.net/projects/udis86/files/udis86/1.7/udis86-1.7.2.tar.gz
+tar -xzf udis86-1.7.2.tar.gz
+cd udis86-1.7.2
+./configure --prefix=$TOOLSDIR/udis86-1.7.2 --enable-shared
+make
+make check
+make install
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of UDIS"
+fi
+
+#install LLVM
+
+cd $BASEDIR
+wget http://llvm.org/releases/3.2/llvm-3.2.src.tar.gz
+tar xzf llvm-3.2.src.tar.gz
+  
+export MY_UDIS_INSTALL_DIR=$TOOLSDIR/udis86-1.7.2
+export MY_LLVM_INSTALL_DIR=$TOOLSDIR/dest-llvm-3.2/
+export MY_LLVM_SRC_DIR=$BASEDIR/llvm-3.2.src
+export MY_LLVM_OBJ_DIR=$BASEDIR/llvm-3.2.obj/
+export LD_LIBRARY_PATH=$MY_UDIS_INSTALL_DIR/lib:$LD_LIBRARY_PATH
+export C_INCLUDE_PATH=$MY_UDIS_INSTALL_DIR/include
+export CPATH=$MY_UDIS_INSTALL_DIR/include
+
+# Build release version
+mkdir -p $MY_LLVM_OBJ_DIR/release
+cd $MY_LLVM_OBJ_DIR/release
+
+$MY_LLVM_SRC_DIR/configure --prefix=$MY_LLVM_INSTALL_DIR/release \
+   --enable-shared --enable-targets=x86,x86_64,cpp \
+   --with-udis86=$MY_UDIS_INSTALL_DIR/lib \
+   CFLAGS=-fgnu89-inline
+
+make libs-only
+make install-libs
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of LLVM release"
+fi
+
+# Build debug version
+mkdir -p $MY_LLVM_OBJ_DIR/debug
+cd $MY_LLVM_OBJ_DIR/debug
+
+$MY_LLVM_SRC_DIR/configure --prefix=$MY_LLVM_INSTALL_DIR/debug \
+   --enable-optimized --enable-jit \
+   --enable-debug-runtime --enable-debug-symbols \
+   --enable-shared --enable-targets=x86,x86_64,cpp \
+   --with-udis86=$MY_UDIS_INSTALL_DIR/lib \
+   CFLAGS=-fgnu89-inline
+
+make libs-only
+make install-libs
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of LLVM debug"
+fi
+
+# Build ICU
+cd $BASEDIR
+wget http://download.icu-project.org/files/icu4c/4.4/icu4c-4_4-src.tgz
+tar -xzf icu4c-4_4-src.tgz
+cd icu/source
+./configure --with-library-suffix=Nv44 --prefix=$TOOLSDIR/icu4.4/linux64
+make
+make check
+make install
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of ICU"
+fi
+
+# make zookeeper
+cd $BASEDIR
+wget https://archive.apache.org/dist/zookeeper/zookeeper-3.4.5/zookeeper-3.4.5.tar.gz 
+tar -zxf zookeeper-3.4.5.tar.gz
+cd zookeeper-3.4.5/src/c
+./configure --prefix=$TOOLSDIR/zookeeper-3.4.5
+make
+make install
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of ZOOKEEPER"
+fi
+
+# make thrift
+cd $BASEDIR
+wget http://archive.apache.org/dist/thrift/0.9.0/thrift-0.9.0.tar.gz
+tar -xzf thrift-0.9.0.tar.gz
+cd thrift-0.9.0
+./configure --prefix=$TOOLSDIR/thrift-0.9.0 -without-qt
+make
+make install
+if [ $VERBOSE -eq 1 ]; then
+  echo "STEP completed:  installation of THRIFT"
+fi
```

Comment:
Good to know, but for now we are probably fine. I can handle the protobuf requirement inside my script for now - or we can move the code to your script (and also update all of our VM images).